content-security-policy/support/testharness-helper.sub.js

```diff
@@ -20,6 +20,20 @@ function getOrigin() {
   return url.toString();
 }
 
+function getCrossOrigin() {
+  var url = new URL("http://{{domains[天気の良い日]}}:{{ports[http][0]}}/");
+  return url.toString();
+}
+
+function getSecureCrossOrigin() {
+  // Since wptserve spins up servers on non-default port, 'self' matches
+  // http://[host]:[specified-port] and https://[host]:[specified-port], but not
+  // https://[host]:[https-port]. So, we use the http port for this https origin
+  // in order to verify that a secure variant of a non-secure URL matches 'self'.
+  var url = new URL("https://{{domains[天気の良い日]}}:{{ports[http][0]}}");
```

Comment:
I just realized that these helpers are in the `content-security-policy/support` file: let's move them to `content-security-policy/embedded-enforcement/support`. This `https` scheme/`http` port mismatch isn't going to be useful generally, and could cause confusion in the future if someone sees `getSecureCrossOrigin` and thinks "Hey, that sounds useful!"